pyproject.toml

```diff
@@ -9,3 +9,10 @@ line_length = 99
 [tool.ruff]
 line-length = 99
 target-version = "py37"
+select = ["E", "F", "I", "PL", "UP", "RUF", "PTH", "C4", "B", "PIE"]
+
+[tool.ruff.pylint]
+max-args = 100
+max-statements = 100
+max-branches = 40
+max-returns = 15
```

Comment:
have these values been used by use before somewhere? Or how did they come to be?

Reply:
We just ignored them in pylint, could do the same here if you'd like..
https://github.com/python-telegram-bot/python-telegram-bot/blob/d5cdb94c6f7b35c54b754781e4d99168d572df15/setup.cfg#L19-L22